### PR TITLE
Automate Dire Growth feat for Werecreature dedication

### PR DESCRIPTION
**Pull request by ammalagonc**

### CODE REVIEW

File: packs/feats/dire-growth.json [hunk 1]
```diff
@@ -31,7 +31,86 @@
...
+                            "damage": 2,
+                            "reach": 10,
+                            "size": "large"
+                        }
+                    }
+                ],
+                "flag": "enlarge",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.CreatureSize"
+            },
```
Comment (pedrogrullada) [inline packs/feats/dire-growth.json:84]: The Choice Set isn't needed. You are under the effects of enlarge at its base rank.

Comment (ammalagonc) [reply packs/feats/dire-growth.json]: Changes made - check when you get a moment.

Thanks @pedrogrullada :)

File: packs/feats/dire-growth.json [hunk 2]
```diff
@@ -31,7 +31,36 @@
...
+                    "change-shape:animal"
+                ],
+                "uuid": "Compendium.pf2e.conditionitems.Item.Clumsy"
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "item:melee"
+                ],
+                "selector": "strike-damage",
```
Comment (TikaelSol) [inline packs/feats/dire-growth.json:59]:
```suggestion
                "key": "FlatModifier",
                "selector": "melee-strike-damage",
```

Comment (TikaelSol) [reply packs/feats/dire-growth.json:59]: We should fix this on lots of things in fact

**Pull request merged**